lib/index.js

```diff
@@ -422,6 +428,7 @@ S3.prototype.put = function(key, data, headers, callback) {
         if (headers['x-amz-server-side-encryption-aws-kms-key-id']) params.SSEKMSKeyId = headers['x-amz-server-side-encryption-aws-kms-key-id'];
         params.ContentType = headers['Content-Type'];
         if (headers['Expires']) params.Expires = headers['Expires'];
+        if (headers['Cache-Control']) params.CacheControl = headers['Cache-Control'];
```

Comment:
`CacheControl` vs `cacheControl` above? are they the same or should we unify on one capitalization?

Reply:
the `cacheControl` casing matches the format of the rest of tilelive-s3, where traditional camel-case is used for the class variables (ie `sseKmsId`. The pascal-cased `CacheControl` is because this is a param for AWS-SDK